wp-content/plugins/wp20-meetup-events/wp20-meetup-events.js

```diff
@@ -361,6 +361,7 @@ var WP20MeetupEvents = app = ( function( $ ) {
 				name      : events[ markerID ].name,
 				time      : events[ markerID ].time,
 				url       : events[ markerID ].eventUrl,
+				title     : events[ markerID ].group,
```

Comment:
Group makes the most sense when tabbing around the map, as it typically includes the location name.